### PR TITLE
T7336: Install BlueSky skin

### PR DESCRIPTION
**Pull request by R4356th**
Per  T7336.

### CODE REVIEW

File: ManageWikiExtensions.php [hunk 1]
```diff
@@ -1747,6 +1747,14 @@
 		'requires' => [],
 		'section' => 'skins',
 	],
+	'bluesky' => [
+		'name' => 'BlueSky',
+		'linkPage' => 'https://www.mediawiki.org/wiki/Skin:BlueSky',
```
Comment (Universal-Omega) [inline ManageWikiExtensions.php:1752]: Special:MyLanguage

File: extension-list [hunk 2]
```diff
@@ -293,3 +293,4 @@ $IP/skins/erudite/skin.json
 $IP/skins/foreground/skin.json
 $IP/skins/liberty/skin.json
 $IP/skins/pivot/skin.json
+$IP/skins/BlueSky/skin.json
```
Comment (Universal-Omega) [inline extension-list:296]: Should be before "Citizen"

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: ManageWikiExtensions.php (hunk 1 above)
Comment (R4356th) [reply ManageWikiExtensions.php:1752]: That is not necessary as that page has not been marked for translation.

Comment (RhinosF1) [reply ManageWikiExtensions.php:1752]: I'd rather we got it marked for translation

Comment (Universal-Omega) [reply ManageWikiExtensions.php:1752]: Still it's preferable to do that so it wouldn't need to be changed later if it were.

File: extension-list (hunk 2 above)
Comment (R4356th) [reply extension-list:296]: Unrelated to this PR.

Comment (Universal-Omega) [reply extension-list:296]: How so?

File: ManageWikiExtensions.php (hunk 1 above)
Comment (R4356th) [reply ManageWikiExtensions.php:1752]: Anyone volunteering?

File: extension-list (hunk 2 above)
Comment (R4356th) [reply extension-list:296]: Look at the lines above that. The whole file needs work and it does not seem worth the effort.

Comment (Universal-Omega) [reply extension-list:296]: No everything else is in order.

Comment (R4356th) [reply extension-list]: See 7dd66a3.

Comment (Universal-Omega) [reply extension-list]: That's not how it should be ordered though.

Comment (Universal-Omega) [reply extension-list]: Capital comes before lowercase in this file.

So Z would come before a in this instance.

Comment (R4356th) [reply extension-list]: Done, I believe.

Comment (Universal-Omega) [reply extension-list]: Yes, thank you!

File: ManageWikiExtensions.php (hunk 1 above)
Comment (R4356th) [reply ManageWikiExtensions.php]: Done.

Comment (Universal-Omega) [reply ManageWikiExtensions.php]: Thank you!

**Pull request merged**